javascript/javascript1/week1/Ez-Namey.js

```diff
@@ -0,0 +1,8 @@
+const firstWords = [
+    "Easy", "Awsome", "Corporate", "Smart", "Bright", "Hack Your", "Next", "True", "Green", "Pure"
+];
+const secondWords = [
+    "Solutions", "Innovations", "Tech", "Future", "Hub", "Lab", "Nest", "Sphere", "Engine", "Nest"
+];
+let startupName = firstWords[Math.floor(Math.random() * 10)] +" "+secondWords[Math.floor(Math.random() * 10)];
```

Comment:
we could put the random numbers in their own variables.. Also, we can make the startupName a const since we're not gonna change it 👍